### PR TITLE
Add jupyterlab-variableinspector

### PR DESCRIPTION
**Pull request by athornton**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by athornton** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -334,3 +335,4 @@ extra:
     - erykoff
     - ktlim
     - beckermr
+    - athornton
```
Comment (erykoff) [inline recipe/meta.yaml:338]: I don't know what happens if you add a maintainer to just the dev branch ...

Comment (athornton) [reply recipe/meta.yaml:338]: I don't mind taking it out.

Comment (athornton) [reply recipe/meta.yaml]: Done

**Comment by athornton** [comment]
@erykoff Would you please do the merge honors, as I cannot?

**Pull request merged**